src/DataFixtures/ORM/LoadApplicationRequestRunningMateRequestData.php

```diff
@@ -41,6 +42,7 @@ public function load(ObjectManager $manager)
             $municipal1->municipalChiefManagedArea()->getCodes()[2],
         ]);
         $runningMateRequest4->setEmailAddress('damien.schmidt@example.ch');
+        $runningMateRequest2->setAdherent($this->getReference('adherent-14'));
```

Comment:
Is that what you planed to do when using `runningMateRequest2` instead of `runningMateRequest4`?

Reply:
nope, fixed 👍 👀